mne/forward/_make_forward.py

```diff
@@ -465,6 +466,7 @@ def _prepare_for_forward(src, mri_head_t, info, bem, mindist, n_jobs,
         raise RuntimeError('No MEG or EEG channels found.')
 
     # pick out final info
+    info['comps'] = []
```

Comment:
I suspect you'll also need one of these in `read_forward_solution`. If in <= 0.15 someone computed a CTF forward, it will have this field, so I think they will get an error on your `assert(` statement in `inverse.py`. Right?

Reply:
I can certainly add it. I don't think it will ever be needed as from looking at 
`_read_forward_meas_info` https://github.com/mne-tools/mne-python/blob/master/mne/forward/forward.py#L249 It doesn't look like `comps` is ever set from anything.

do you want it to be silent or issue a `RuntimeWarning`

Reply:
You're right -- I just tried creating a CTF forward on `master`, writing to disk, reading it, and `fwd['info']['comps']` does indeed go away. So nothing to do here, I think.

Reply:
you do not need the `+` and I would make it a bit more explanatory "..., this should not affect application of the forward model but it will change the model if it's written back to disk" or so, to give people a sense of what the consequences are

Reply:
Ok no change.
Do you still want an update to the `api` section?

Reply:
I *think* that we don't need it anymore